src/python/pants/backend/python/rules/python_sources.py

```diff
@@ -4,9 +4,6 @@
 from dataclasses import dataclass
```

Comment:
The docstring is out of date in this file. It should remove all mention of `__init__.py`.